src/components/leaderboard/Leaderboard.js

```diff
@@ -65,8 +75,7 @@ export default function Leaderboard({ filteredPlayers }) {
         <TableBody>
           {displayPlayers.map((player) => (
             <TableRow
-              key={player.name}
-              sx={{ '&:last-child td, &:last-child th': { border: 5 } }}
+              key={`${player.name}_${player.rank}`}
```

Comment:
The key needs to be unique, so I just concatenated the name with the rank, which is then unique.